afrc/src/afrc/Search/SearchPage.vue

```diff
@@ -201,8 +208,15 @@ onMounted(async () => {
                     :basemaps="basemaps"
                     :overlays="overlays"
                     :sources="sources"
-                    :include-drawer="false"
+                    :include-drawer="true"
                     :popup-enabled="false"
+                    @drawn-features-updated="
+                    (features) => {
+                        updateDrawnFeaturesGeometry(
+                            features,
+                        );
+                    }
+                "
```

Comment:
this is more direct and should provide the same results
```suggestion
                    @drawn-features-updated="updateDrawnFeaturesGeometry"
```

Reply:
Thanks, yeah - that's much cleaner